### ISSUE TITLE
Add configuration file for default settings

### ISSUE THREAD

**Issue by bvaisvil** [bvaisvil/zenith#71]
When a user arranges the section sizes to what they want, it would be nice to be able to save this size in a configuration. In addition, perhaps we could add support for customizing the colors of various charts.

**Comment by JacobTravers** [comment]
It would also be nice to have configurable keybindings, so we could add {vim, emacs, kakoune, etc}-like keybindings

**Comment by cw1nte** [comment]
I agree, command line switches or a .zenith config file would both save time...

**Comment by cw1nte** [comment]
or even saving a replay at startup this keys sequence??
